Week1/homework/js-exercises/LogNumber.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+var numberX;
+console.log('Value of x is his age');
+console.log(numberX);
+numberX = 32;
```

Comment:
`let` instead of `var` here please. Also, you were expected to write in comments or console log what you think the value of `numberX` would be before you do `console.log(numberX)`. Could you please correct it?